tests/test_api/test_item.py

```diff
@@ -19,6 +26,38 @@ def test_get_item_missing(self, test_client, item):
         check_valid_header_type(r.headers)
         assert r.status_code == 404
 
+    def test_put_item_upvote(self, test_client, user, item):
+        self.login(test_client, user.email, user.password)
+
+        upvote = dict(vote=1)
```

Comment:
Sounds good, we should still see if we can get the api to validate the integer is within a range. We don't want a malicious user trying to send a bigger number.